### PR TITLE
Added landing page

### PR DESCRIPTION
**Pull request by jeetsdev**
Added Landing page

- Navbar
- Landing section
- Shop by categories 
- New arrivals
- Footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+                    </div>
+                    <!-- <div class="category__sec category__sec-couple center__flex">
+                        <div class="categroy__sec-image">
+                            <img src="assets/made4dev-coffee-to-code-tee.png" alt="" class="image-res">
+                        </div>
+                        <div class="overlay__sec-text new-arr center__flex">
+                            <p class="center__flex text-mid"> Couple </p>
+                        </div>
+                    </div> -->
+
```
Comment (romabulani) [inline index.html]: You should remove comments if not required

File: index.html [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+                    <div class="category__sec center__flex">
+                        <div class="categroy__sec-unisex border__rad-4px">
+                            <div class="categroy__sec-image">
+                                <img src="assets/t-shirt-2.png" alt="" class="image-res">
+                            </div>
+                            <div class="overlay__sec-text new-arr center__flex">
+                                <p class="center__flex text-mid"> Unisex </p>
+                            </div>
+                        </div>
+                    </div>
```
Comment (romabulani) [inline index.html]: Always give values to alt attribute

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

**Review by narharikale — APPROVED** [review]
UI is lit 🔥..

**Pull request merged**